### ISSUE TITLE
[changelog] Non-existent contributor appears and links to pull requests appear twice

### ISSUE THREAD

**Issue by g3n35i5** [jreleaser/jreleaser#1205]
### Task List

- [x] Steps to reproduce provided
- [x] Stacktrace (if present) provided
- [ ] Example that reproduces the problem (link to git repository is ideal)
- [x] Full description of the issue provided (see below)

Unfortunately I can't publish the source code and commit messages because it is a private repository. However, I assume that the logs (which I have blacked out a bit) are sufficient.

### Steps to Reproduce

1. Publish a release to GitHub with the `jreleaser.yml` command and configuration file shown below

**Command**
```bash
export JRELEASER_PROJECT_NAME=FOOBAR
export JRELEASER_PROJECT_VERSION=2.0.0
export JRELEASER_GITHUB_TOKEN=<SECRET>
export JRELEASER_SKIP_TAG=true
jreleaser release
```

**Configuration file**
```yml
release:
  github:
    owner: <REMOVED>
    tagName: "{{projectVersion}}"
    changelog:
      formatted: ALWAYS
      preset: conventional-commits
      contributors:
        format: "- {{contributorName}}{{#contributorUsernameAsLink}} ({{.}}){{/contributorUsernameAsLink}}"

```

### Expected Behaviour

Since I am the only contributor for this release, I'd expect the contributors section to look like this:
```
We'd like to thank the following people for their contributions:
- Jan-Frederik Schmidt
```

### Actual Behaviour

However, the user @betterqualityassuranceuser is listed as contributor. As you can see, the only contributor in all commits is me (`Jan-Frederik Schmidt <jan-frederik.schmidt@vorausrobotik.com>`)
```
We'd like to thank the following people for their contributions:
- GitHub ([@betterqualityassuranceuser](https://github.com/betterqualityassuranceuser))
- Jan-Frederik Schmidt
```

Moreover, the default GitHub behavior is to put a link to PR into the squash commit message, e.g. `feat: Foobar (#4242424)`. JReleaser somehow duplicates this reference to `(#4242424), closes #4242424`

### Environment Information

- **Operating System**: 5.10.0-19-amd64 Debian 5.10.149-2 (2022-10-21) x86_64 GNU/Linux
- **JReleaser Version:** 1.4.0
- **JDK Version:** 11.0.16+8-1~deb11u1

<details>
  <summary>(Stripped) git log</summary>
  
  ```
commit 7df1e4097e7f084fee74adef4b9d2fd838305cf2
Author: Jan-Frederik Schmidt <jan-frederik.schmidt@vorausrobotik.com>
Date:   Thu Feb 9 11:13:32 2023 +0100

    build: <REMOVED> (#15)

commit 7df0fe94a6c2c44565dd9df1ae7548057559cded
Author: Jan-Frederik Schmidt <jan-frederik.schmidt@vorausrobotik.com>
Date:   Thu Feb 9 11:07:30 2023 +0100

    ci: <REMOVED> (#14)

commit 9a4504451a923c174556ab1de19cc59a449e0651
Author: Jan-Frederik Schmidt <jan-frederik.schmidt@vorausrobotik.com>
Date:   Thu Feb 9 10:05:10 2023 +0100

    ci!: <REMOVED>

commit df0e2f174658f100a11de7d8f98bdf3740dedc50
Author: Jan-Frederik Schmidt <jan-frederik.schmidt@vorausrobotik.com>
Date:   Thu Feb 9 10:04:55 2023 +0100

    build: <REMOVED>

commit 4c2641542167faf41c2fdc1f0dba52b81f85ba08
Author: Jan-Frederik Schmidt <jan-frederik.schmidt@vorausrobotik.com>
Date:   Thu Feb 9 08:08:57 2023 +0100

    ci: <REMOVED> (#12)

commit 92f1e3be913bf9b34de480a6aab89df632fa4b3d
Author: Jan-Frederik Schmidt <jan-frederik.schmidt@vorausrobotik.com>
Date:   Mon Jan 30 15:37:14 2023 +0100

    fix: <REMOVED> (#11)

commit 9984f097a20d1e0ed77c1ab4c08b9b06fb02ee4f
Author: Jan-Frederik Schmidt <jan-frederik.schmidt@vorausrobotik.com>
Date:   Mon Jan 30 09:52:14 2023 +0100

    fix: <REMOVED> (#10)

commit 6b11bfcecb2b138910e45887db011b3498c926b3
Author: Jan-Frederik Schmidt <jan-frederik.schmidt@vorausrobotik.com>
Date:   Wed Jan 25 10:26:48 2023 +0100

    build: <REMOVED>
```
</details>

<details>
  <summary>JReleaser trace.log</summary>
  
  ```
[INFO]  JReleaser 1.4.0
------------------------------------------------------------
jreleaser 1.4.0
------------------------------------------------------------
Build time:   2022-12-29 15:08:38Z
Revision:     b977b66baed1e4761af054c5d747a0804f457252
JVM:          17.0.4 (Debian 17.0.4+8-Debian-1deb11u1)
------------------------------------------------------------
[INFO]  Configuring with jreleaser.yml
[INFO]    - basedir set to <REMOVED>
[INFO]    - outputdir set to <REMOVED>/out/jreleaser
[INFO]  Reading configuration
[DEBUG]   - basedir set to <REMOVED>
[DEBUG]   - baseOutputDirectory set to <REMOVED>/out
[DEBUG]   - outputDirectory set to <REMOVED>/out/jreleaser
[DEBUG]   - checksumDirectory set to <REMOVED>/out/jreleaser/checksums
[DEBUG]   - signaturesDirectory set to <REMOVED>/out/jreleaser/signatures
[DEBUG]   - prepareDirectory set to <REMOVED>/out/jreleaser/prepare
[DEBUG]   - packageDirectory set to <REMOVED>/out/jreleaser/package
[DEBUG]   - downloadDirectory set to <REMOVED>/out/jreleaser/download
[DEBUG]   - assembleDirectory set to <REMOVED>/out/jreleaser/assemble
[DEBUG]   - artifactsDirectory set to <REMOVED>/out/jreleaser/artifacts
[INFO]  git-root-search set to false
[INFO]  Loading variables from /home/localuser/.jreleaser/config.properties
[WARN]  Variables source /home/localuser/.jreleaser/config.properties does not exist
[INFO]  Validating configuration
[INFO]  Strict mode set to false
[DEBUG] adjusting distributions with assemblies
[DEBUG]   [validation] --== ASSEMBLE ==--
[DEBUG]   [validation] hooks
[DEBUG]   [validation] hooks.command
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] project
[DEBUG]   [validation] download
[DEBUG]   [validation] assemble
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] signing
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] release
[DEBUG]   [validation] release.github
[DEBUG]   [validation] checksum
[DEBUG]   [validation] deploy
[DEBUG]   [validation] deploy.maven
[DEBUG]   [validation] upload
[DEBUG]   [validation] announce
[DEBUG]   [validation] - disabled
[DEBUG]   [postvalidation] project
[DEBUG]   [postvalidation] assemble
[DEBUG]   [postvalidation] assemble.jpackage
[DEBUG]   [resolution] assemble
[DEBUG]   [validation] --== FULL ==--
[DEBUG]   [validation] hooks
[DEBUG]   [validation] hooks.command
[DEBUG]   [validation] project
[DEBUG]   [validation] download
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] assemble
[DEBUG]   [validation] signing
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] release
[DEBUG]   [validation] release.github
[DEBUG]   [validation] checksum
[DEBUG]   [validation] deploy
[DEBUG]   [validation] deploy.maven
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] upload
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] packagers
[DEBUG]   [validation] files
[DEBUG]   [validation] announce
[DEBUG]   [validation] announce.article
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] announce.discussions
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] announce.discord
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] announce.discourse
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] announce.gitter
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] announce.googleChat
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] announce.http
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] announce.mail
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] announce.mastodon
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] announce.mattermost
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] announce.sdkman
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] announce.slack
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] announce.teams
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] announce.telegram
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] announce.twitter
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] announce.webhooks
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] announce.zulip
[DEBUG]   [validation] - disabled
[DEBUG]   [validation] - disabled
[DEBUG]   [postvalidation] project
[DEBUG]   [postvalidation] assemble
[DEBUG]   [postvalidation] assemble.jpackage
[DEBUG]   [postvalidation] distributions
environment:
 
hooks:
    enabled: false
    active: never
    command:
        enabled: false
        active: never
 
project:
    name: <REMOVED>
    version: 2.0.0
    versionPattern: semver
    snapshot:
        enabled: false
        pattern: .*-SNAPSHOT
        label: early-access
        fullChangelog: false
    stereotype: none
    links:
        bugTracker: https://{{repoHost}}/{{repoOwner}}/{{repoName}}/issues
        vcsBrowser: https://{{repoHost}}/{{repoOwner}}/{{repoName}}
    extraProperties:
        projectVersionMajor: 2
        projectVersionMinor: 0
        projectVersionPatch: 0
        projectVersionNumber: 2.0.0
        projectVersionWithUnderscores: 2_0_0
        projectVersionWithDashes: 2-0-0
        projectVersionNumberWithUnderscores: 2_0_0
        projectVersionNumberWithDashes: 2-0-0
        projectEffectiveVersionWithUnderscores: 2_0_0
        projectEffectiveVersionWithDashes: 2-0-0
 
platform:
 
release:
    github:
        enabled: true
        host: github.com
        owner: <REMOVED>
        name: <REMOVED>
        username: <REMOVED>
        token: ************
        artifacts: true
        files: true
        checksums: true
        signatures: true
        repoUrl: https://{{repoHost}}/{{repoOwner}}/{{repoName}}
        repoCloneUrl: https://{{repoHost}}/{{repoOwner}}/{{repoName}}.git
        commitUrl: https://{{repoHost}}/{{repoOwner}}/{{repoName}}/commits
        srcUrl: https://{{repoHost}}/{{repoOwner}}/{{repoName}}/blob/{{repoBranch}}
        downloadUrl: https://{{repoHost}}/{{repoOwner}}/{{repoName}}/releases/download/{{tagName}}/{{artifactFile}}
        releaseNotesUrl: https://{{repoHost}}/{{repoOwner}}/{{repoName}}/releases/tag/{{tagName}}
        latestReleaseUrl: https://{{repoHost}}/{{repoOwner}}/{{repoName}}/releases/latest
        issueTrackerUrl: https://{{repoHost}}/{{repoOwner}}/{{repoName}}/issues
        tagName: {{projectVersion}}
        releaseName: Release {{tagName}}
        branch: main
        commitAuthor:
            name: jreleaserbot
            email: jreleaser@kordamp.org
        sign: false
        skipTag: true
        skipRelease: false
        overwrite: false
        update:
            enabled: false
        apiEndpoint: https://api.github.com
        connectTimeout: 20
        readTimeout: 60
        changelog:
            enabled: true
            append:
                enabled: false
            links: false
            skipMergeCommits: false
            sort: DESC
            formatted: always
            preset: conventional-commits
            format: - {{commitShortHash}} {{#commitIsConventional}}{{#conventionalCommitIsBreakingChange}}ðŸš¨ {{/conventionalCommitIsBreakingChange}}{{#conventionalCommitScope}}**{{conventionalCommitScope}}**: {{/conventionalCommitScope}}{{conventionalCommitDescription}}{{#conventionalCommitBreakingChangeContent}} - *{{conventionalCommitBreakingChangeContent}}*{{/conventionalCommitBreakingChangeContent}}{{/commitIsConventional}}{{^commitIsConventional}}{{commitTitle}}{{/commitIsConventional}}{{#commitHasIssues}}, closes{{#commitIssues}} {{issue}}{{/commitIssues}}{{/commitHasIssues}}
            categoryTitleFormat: ## {{categoryTitle}}
            contributorsTitleFormat: ## Contributors
            content: 
## Changelog

{{changelogChanges}}
    {{changelogContributors}}
            hide:
            contributors:
                enabled: true
                format: - {{contributorName}}{{#contributorUsernameAsLink}} ({{.}}){{/contributorUsernameAsLink}}
            categories:
                category 0:
                    key: merge
                    title: ðŸ”€ Merge
                    labels:
                        merge
                    order: 0
                category 1:
                    key: features
                    title: ðŸš€ Features
                    labels:
                        feat
                    order: 10
                category 2:
                    key: fixes
                    title: ðŸ› Fixes
                    labels:
                        fix
                    order: 20
                category 3:
                    key: changes
                    title: ðŸ”„ï¸ Changes
                    labels:
                        revert
                        style
                        perf
                        refactor
                    order: 30
                category 4:
                    key: tasks
                    title: ðŸ§° Tasks
                    labels:
                        chore
                    order: 40
                category 5:
                    key: build
                    title: ðŸ›   Build
                    labels:
                        test
                        build
                        ci
                    order: 50
                category 6:
                    key: docs
                    title: ðŸ“ Documentation
                    labels:
                        docs
                    order: 60
            labelers:
                labeler 0:
                    label: merge
                    title: Merge pull
                    order: 0
                labeler 1:
                    label: merge
                    title: Merge branch
                    order: 5
                labeler 2:
                    label: build
                    title: regex:^(?:build(?:\(.*\))?!?):\s.*
                    order: 10
                labeler 3:
                    label: chore
                    title: regex:^(?:chore(?:\(.*\))?!?):\s.*
                    order: 20
                labeler 4:
                    label: ci
                    title: regex:^(?:ci(?:\(.*\))?!?):\s.*
                    order: 30
                labeler 5:
                    label: docs
                    title: regex:^(?:docs(?:\(.*\))?!?):\s.*
                    order: 40
                labeler 6:
                    label: feat
                    title: regex:^(?:feat(?:\(.*\))?!?):\s.*
                    order: 50
                labeler 7:
                    label: fix
                    title: regex:^(?:fix(?:\(.*\))?!?):\s.*
                    order: 60
                labeler 8:
                    label: perf
                    title: regex:^(?:perf(?:\(.*\))?!?):\s.*
                    order: 70
                labeler 9:
                    label: refactor
                    title: regex:^(?:refactor(?:\(.*\))?!?):\s.*
                    order: 80
                labeler 10:
                    label: revert
                    title: regex:^(?:revert(?:\(.*\))?!?):\s.*
                    order: 90
                labeler 11:
                    label: style
                    title: regex:^(?:style(?:\(.*\))?!?):\s.*
                    order: 100
                labeler 12:
                    label: test
                    title: regex:^(?:test(?:\(.*\))?!?):\s.*
                    order: 110
        milestone:
            name: {{tagName}}
            close: true
        issues:
            enabled: false
            comment: Released in {{tagName}} -> {{releaseNotesUrl}}
            label:
                name: released
                color: #FF0000
                description: Issue has been released
        prerelease:
            enabled: false
        draft: false
        releaseNotes:
            enabled: false
 
checksum:
    name: checksums.txt
    individual: false
    algorithms:
        SHA_256
    files: true
 
signing:
    enabled: false
    active: never
    armored: false
    artifacts: true
    files: true
    checksums: true
    passphrase: **unset**
    publicKey: **unset**
    secretKey: **unset**
 
announce:
    enabled: false
    active: never
    article:
        enabled: false
        active: never
        commitAuthor:
        repository:
            enabled: false
            name: article
            token: **unset**
    discord:
        enabled: false
        active: never
        webhook: **unset**
    discourse:
        enabled: false
        active: never
        apiKey: **unset**
        username: **unset**
    discussions:
        enabled: false
        active: never
    gitter:
        enabled: false
        active: never
        webhook: **unset**
    googlechat:
        enabled: false
        active: never
        webhook: **unset**
    smtp:
        enabled: false
        active: never
        auth: false
        password: **unset**
    mastodon:
        enabled: false
        active: never
        accessToken: **unset**
    mattermost:
        enabled: false
        active: never
        webhook: **unset**
        structuredMessage: true
    sdkman:
        enabled: false
        active: never
        consumerKey: **unset**
        consumerToken: **unset**
    slack:
        enabled: false
        active: never
        webhook: **unset**
        token: **unset**
    teams:
        enabled: false
        active: never
        webhook: **unset**
    telegram:
        enabled: false
        active: never
        token: **unset**
        chatId: **unset**
    twitter:
        enabled: false
        active: never
        consumerKey: **unset**
        consumerSecret: **unset**
        accessToken: **unset**
        accessTokenSecret: **unset**
    zulip:
        enabled: false
        active: never
        apiKey: **unset**
 
packagers:
    appimage:
        enabled: false
        active: never
        continueOnError: false
        commitAuthor:
            name: jreleaserbot
            email: jreleaser@kordamp.org
        requiresTerminal: false
        repository:
            enabled: true
            active: release
            owner: <REMOVED>
            name: appimage
            tagName: {{projectVersion}}
            branch: HEAD
            username: <REMOVED>
            token: ************
    asdf:
        enabled: false
        active: never
        continueOnError: false
        commitAuthor:
            name: jreleaserbot
            email: jreleaser@kordamp.org
        repository:
            enabled: true
            active: release
            owner: <REMOVED>
            name: asdf
            tagName: {{projectVersion}}
            branch: HEAD
            username: <REMOVED>
            token: ************
    brew:
        enabled: false
        active: never
        continueOnError: false
        commitAuthor:
            name: jreleaserbot
            email: jreleaser@kordamp.org
        multiPlatform: false
        tap:
            enabled: true
            active: release
            owner: <REMOVED>
            name: homebrew-tap
            tagName: {{projectVersion}}
            branch: HEAD
            username: <REMOVED>
            token: ************
        cask:
            enabled: false
    chocolatey:
        enabled: false
        active: never
        continueOnError: false
        commitAuthor:
            name: jreleaserbot
            email: jreleaser@kordamp.org
        apiKey: **unset**
        remoteBuild: false
        bucket:
            enabled: true
            active: release
            owner: <REMOVED>
            name: chocolatey-bucket
            tagName: {{projectVersion}}
            branch: HEAD
            username: <REMOVED>
            token: ************
    docker:
        enabled: false
        active: never
        useLocalArtifact: true
        buildx:
            enabled: false
        commitAuthor:
            name: jreleaserbot
            email: jreleaser@kordamp.org
        repository:
            enabled: true
            active: release
            owner: <REMOVED>
            name: docker
            tagName: {{projectVersion}}
            branch: HEAD
            username: <REMOVED>
            token: ************
            versionedSubfolders: false
        continueOnError: false
    flatpak:
        enabled: false
        active: never
        continueOnError: false
        commitAuthor:
            name: jreleaserbot
            email: jreleaser@kordamp.org
        repository:
            enabled: true
            active: release
            owner: <REMOVED>
            name: flatpak
            tagName: {{projectVersion}}
            branch: HEAD
            username: <REMOVED>
            token: ************
    gofish:
        enabled: false
        active: never
        continueOnError: false
        commitAuthor:
            name: jreleaserbot
            email: jreleaser@kordamp.org
        repository:
            enabled: true
            active: release
            owner: <REMOVED>
            name: <REMOVED>-fish-food
            tagName: {{projectVersion}}
            branch: HEAD
            username: <REMOVED>
            token: ************
    jbang:
        enabled: false
        active: never
        continueOnError: false
        commitAuthor:
            name: jreleaserbot
            email: jreleaser@kordamp.org
        catalog:
            enabled: true
            active: always
            owner: <REMOVED>
            name: jbang-catalog
            tagName: {{projectVersion}}
            branch: HEAD
            username: <REMOVED>
            token: ************
    macports:
        enabled: false
        active: never
        continueOnError: false
        commitAuthor:
            name: jreleaserbot
            email: jreleaser@kordamp.org
        repository:
            enabled: true
            active: release
            owner: <REMOVED>
            name: macports
            tagName: {{projectVersion}}
            branch: HEAD
            username: <REMOVED>
            token: ************
    scoop:
        enabled: false
        active: never
        continueOnError: false
        commitAuthor:
            name: jreleaserbot
            email: jreleaser@kordamp.org
        bucket:
            enabled: true
            active: release
            owner: <REMOVED>
            name: scoop-<REMOVED>
            tagName: {{projectVersion}}
            branch: HEAD
            username: <REMOVED>
            token: ************
    sdkman:
        enabled: false
        active: never
        continueOnError: false
        connectTimeout: 20
        readTimeout: 60
        consumerKey: **unset**
        consumerToken: **unset**
    snap:
        enabled: false
        active: never
        continueOnError: false
        commitAuthor:
            name: jreleaserbot
            email: jreleaser@kordamp.org
        base: core20
        grade: stable
        confinement: strict
        remoteBuild: false
        snap:
            enabled: true
            active: release
            owner: <REMOVED>
            name: snap
            tagName: {{projectVersion}}
            branch: HEAD
            username: <REMOVED>
            token: ************
    spec:
        enabled: false
        active: never
        continueOnError: false
        commitAuthor:
            name: jreleaserbot
            email: jreleaser@kordamp.org
        repository:
            enabled: true
            active: release
            owner: <REMOVED>
            name: <REMOVED>-spec
            tagName: {{projectVersion}}
            branch: HEAD
            username: <REMOVED>
            token: ************
 
download:
    enabled: false
    active: never
 
assemble:
    enabled: false
    active: never
 
deploy:
    enabled: false
    active: never
    maven:
        enabled: false
        active: never
 
upload:
    enabled: false
    active: never
 
[INFO]  Project version set to 2.0.0
[INFO]  Release is not snapshot
[INFO]  Timestamp is 2023-02-09T11:32:05.763718977+01:00
[INFO]  HEAD is at 7df1e40
[INFO]  Platform is linux-x86_64
[DEBUG] loading extension default-jreleaser-mustache:jreleaser.shadow.org.jreleaser.extensions.internal.mustache.DefaultMustacheExtension
[DEBUG] adding jreleaser.shadow.org.jreleaser.extensions.internal.mustache.DefaultMustacheExtensionPoint from default-jreleaser-mustache extension
[INFO]  dry-run set to false
[INFO]  Generating changelog
[DEBUG] resolving commits
[DEBUG] looking for tag that matches '2.0.0'
[DEBUG] looking for a tag before '2.0.0' that matches '.*'
[DEBUG] found tag 1.0.0
[DEBUG] sorting commits DESC
[DEBUG] looking up user for GitHub <noreply@github.com>
[DEBUG] looking up user for Jan-Frederik Schmidt <jan-frederik.schmidt@<REMOVED>.com>
[INFO]  Storing changelog: out/jreleaser/release/CHANGELOG.md
[DEBUG] ## Changelog

## ðŸ› Fixes
- 92f1e3b <REMOVED> (#11), closes #11
- 9984f09 <REMOVED> (#10), closes #10

## ðŸ›   Build
- 7df1e40 <REMOVED> (#15), closes #15
- 7df0fe9 <REMOVED> (#14), closes #14
- 9a45044 <REMOVED>
- df0e2f1 <REMOVED>
- 4c26415 <REMOVED> (#12), closes #12
- 6b11bfc <REMOVED>


## Contributors
We'd like to thank the following people for their contributions:
- GitHub ([@betterqualityassuranceuser](https://github.com/betterqualityassuranceuser))
- Jan-Frederik Schmidt
[INFO]  Calculating checksums
[INFO]    [checksum] No files configured for checksum. Skipping
[INFO]  Signing files
[INFO]    [sign] Signing is not enabled. Skipping
[INFO]  Deploying is not enabled. Skipping
[INFO]  Uploading is not enabled. Skipping
[INFO]  Releasing to https://github.com/<REMOVED>/<REMOVED>
[DEBUG] fetching all branches on <REMOVED>/<REMOVED>
[DEBUG] looking up release with tag 2.0.0 at repository <REMOVED>/<REMOVED>
[DEBUG] fetching release on <REMOVED>/<REMOVED> with tag 2.0.0
[DEBUG] release 2.0.0 does not exist
[DEBUG] creating release 2.0.0
[DEBUG] creating release on <REMOVED>/<REMOVED> with tag 2.0.0
[DEBUG] lookup open milestone '2.0.0' on <REMOVED>/<REMOVED>
[DEBUG] lookup repository <REMOVED>/<REMOVED>
[INFO]  Writing output properties to out/jreleaser/output.properties
[INFO]  JReleaser succeeded after 4.427 s
```
</details>

**Comment by aalmiray** [comment]
Issue number duplicated in rendered changelog -> https://github.com/jreleaser/jreleaser/issues/1189

The user/email `GitHub <noreply@github.com>` should be skipped no matter what. This requires an update to the code.

**Comment by g3n35i5** [comment]
Wow, that was fast. Sorry, I didn't see #1189. Thanks for your fast fix!

**Comment by aalmiray** [comment]
Released in v1.5.0 -> https://github.com/jreleaser/jreleaser/releases/tag/v1.5.0
